dair_pll/tensor_utils.py

```diff
@@ -162,6 +162,35 @@ def skew_symmetric(vectors: Tensor) -> Tensor:
     return torch.stack((row_1, row_2, row_3), -2)
 
 
+def nonskew_symmetric(vectors: Tensor) -> Tensor:
```

Comment:
Naming is a little clunky. The space is skew symmetric matrices has a 3-dimensional basis, so the conversion in that case is a little more clear; also "nonskew symemtric" seems to just suggest "symmetric" and leaves out "zero diagonal". Suggest refactor to `symmetric_offdiagonal` or something similar

Reply:
Great suggestion -- I adopted exactly that suggested name and updated the documentation slightly in [this commit](https://github.com/DAIRLab/dair_pll/pull/4/commits/8db8ec87cbf382d33a38f32d077391f8e47e125d).